### PR TITLE
Document History access fields

### LINKED ISSUE #1275: Document sequence attributes of the history object
```python
self.tss = CommandField('ts', self)
self.inps = CommandField('inp', self)
self.outs = CommandField('out', self)
self.rtns = CommandField('rtn', self)
```

It turns out these are supposed to be the public interface to the history object but they scared me away because they're missing in [the documentation](http://xon.sh/api/history.html#xonsh.history.History)

### PR DESCRIPTION
**Pull request by astronouth7303**
Fixes #1275.

### CODE REVIEW

File: xonsh/history.py [hunk 1]
```diff
@@ -505,7 +505,22 @@ def _hist_show(ns=None, hist=None, start_index=None, end_index=None,
 # Interface to History
 #
 class History(object):
-    """Xonsh session history."""
+    """Xonsh session history.
+
+    History is accessed through several sequence-like fields:
+
+    rtns : sequence of ints
```
Comment (scopatz) [inline xonsh/history.py]: I think that these should be in an Attributes section as per numpydoc https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#class-docstring

**Comment by scopatz** [comment]
Thanks for putting this in @astronouth7303!

**Comment by astronouth7303** [comment]
Clean up.

**Comment by scopatz** [comment]
Thanks a million!

**Pull request merged**